### ISSUE TITLE
REST API Specifications

### ISSUE THREAD

**Issue by ate47** [the-qa-company/qEndpoint#239]
### Part of the endpoint? (leave empty if you don't know)

- [X] Backend (hdt-qs-backend)
- [X] Frontend (hdt-qs-frontend)
- [ ] Benchmark (hdt-qs-benchmark)
- [ ] Other

### Description of the request

We don't have specs for the REST API of the endpoint

### Do I want to contribute to make it?

Maybe

**Comment by namedgraph** [comment]
@ate47 if you mean the `/api/endpoint/sparql` endpoint mentioned in the README, then it's the [SPARQL 1.1 Protocol endpoint](https://www.w3.org/TR/sparql11-protocol/).

**Comment by ate47** [comment]
No, I mean all the /api/** content, like the logs, reindex, etc.
